tests/test_jwt.py

```diff
@@ -328,5 +325,5 @@ def setUp(self):
 class TestHasOpenSSLFlag(unittest2.TestCase):
 
     def test_true(self):
-        self.assertEqual(True, HAS_OPENSSL)
-        self.assertEqual(True, HAS_CRYPTO)
+        self.assertEqual(True, client.HAS_OPENSSL)
```

Comment:
Also possibly in a future cleanup: `assertTrue` or `assertIs` (depending on the semantics of the system under test).